ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -587,7 +589,8 @@ Loader {
             }
 
             replyDetails: StatusMessageDetails {
-                messageText:  delegate.replyMessage ? delegate.replyMessage.messageText : ""
+                messageText:  delegate.replyMessage ? delegate.replyMessage.messageText
+                                                    : qsTr("&lt;deleted&gt;")
```

Comment:
the `qsTr()` needs some context; "deleted" is an adjective but the translator wouldn't know what the subject is

Reply:
why? What do you mean? any example?

Reply:
https://doc.qt.io/qt-5/qtquick-internationalization.html#2-add-context-for-the-translator

Reply:
"deleted message" would've been sufficient, but thanks :)